src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -151,11 +151,12 @@ class WorkspaceReimburseView extends React.Component {
             rateValue: numValue.toFixed(3),
         });
 
-        Policy.setCustomUnitRate(this.props.policyID, this.state.unitID, {
-            customUnitRateID: this.state.rateID,
-            name: this.state.rateName,
-            rate: numValue.toFixed(3) * 100,
-        }, null);
+        const distanceCustomUnit = _.find(lodashGet(this.props, 'policy.customUnits', {}), unit => unit.name === 'Distance');
+        const currentCustomUnitRate = lodashGet(distanceCustomUnit, ['rates', this.state.unitRateID], {});
+        Policy.setCustomUnitRate(this.props.policyID, currentCustomUnitRate, this.state.unitID, {
+            ..._.omit(currentCustomUnitRate, 'rate'),
```

Comment:
The `omit` is unnecessary. By doing the spread first, followed by the updated `rate`, the second `rate` will just overwrite the contents of what was in `currentCustomUnitRate.rate`